CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 * Support for handling new elements (location and table parameters) coming in from Apiary events.
 * Support for performing metadata only sync in alter table and alter partition event.
 
+### Changed
+* Extracted Apiary sqs receiver logic to library in [`apiary-extensions`](https://github.com/ExpediaInc/apiary-extensions).
```

Comment:
You should link to the full `apiary-receivers` location, you can put the full path in here just make sure we merge the other PR first.